src/main/client/scene/game-scene.ts

```diff
@@ -45,8 +61,23 @@ export class GameScene extends Scene {
          });
          this.mapComponent.setMapSprite(this.mapSprite);
       });
-      this.mapComponent.updated$.subscribe(() => {
-         this.mapSprite && this.mapSprite.update();
+      this.mapComponent.updated$.subscribe(() => this.mapSprite && this.mapSprite.update());
+      this.created$.pipe(switchMap(() => this.otherPlayersComponent.added$)).subscribe((player) => {
+         const sprite = new OtherPlayerSprite(this, player);
+         this.otherPlayers.set(player.id, sprite);
+         // TODO: Cleanup
```

Comment:
Not sure it is the best way doing the subscriptions here. We should unscubsribe somehow when the player gets removed.